tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/package.scala

```diff
@@ -1,22 +1,217 @@
 package at.forsyte.apalache.tla
 
 import at.forsyte.apalache.tla.lir._
-import at.forsyte.apalache.tla.lir.oper.TlaOper
+import at.forsyte.apalache.tla.lir.oper._
 import scalaz._
 
 import scala.language.implicitConversions
 
 /**
- * The key definitions related to the typed builder. The most important ones for the users of this API are the methods:
+ * This package defines key types and conversions related to [[ScopedBuilder]].
  *
- *   - an implicit conversion [[typecomp#build]] that converts a builder state to its final form, e.g., to `TlaEx`.
- *   - an implicit conversion [[typecomp#liftBuildToSeq]] that applies `build` to a sequence.
+ * =A brief introduction to [[ScopedBuilder]]=
```

Comment:
Maybe we can rename this a bit to communicate that this is not user-facing documentation.

```suggestion
 * =A brief introduction to [[ScopedBuilder]] development=
```